openseadragon.module

```diff
@@ -133,6 +142,8 @@ function template_preprocess_openseadragon_iiif_manifest_block(&$variables) {
     return;
   }
 
+  $viewer_settings['sequenceMode'] = count($tile_sources) > 1 && !$viewer_settings['collectionMode'];
```

Comment:
Do we need to set this in both the tilesource formatter (line 102) and here?

Reply:
We have it set in both.  In theory, if you have a multi-valued field, you'd have more than one to show and would want sequence or collection mode enabled.

It's not totally neccessary, but figured while I was at it I'd put it in there.

Reply:
But would each field show its own OSD block? Maybe I am misunderstanding but it would seem if you set the OSD setting once, then you shouldn't have to set it again.